src/App.tsx

```diff
@@ -9,7 +7,7 @@ import RulesMarkdown from './rules.md'
 import ThanksMarkdown from './thanks.md'
 import ReactMarkdown from 'react-markdown'
 import remarkGfm from 'remark-gfm'
-
+import React from 'react'
```

Comment:
Tarviiks tätä importtia?

Reply:
Jos tän poistaa niin saa 'React' refers to a UMD global, but the current file is a module. Consider adding an import instead.ts(2686)